docs/qsimcirq_tutorial.ipynb

```diff
@@ -0,0 +1,636 @@
+      },
+      "source": [
+        "import time\n",
+        "\n",
+        "qubits = [cirq.GridQubit(i,j) for i in range(4) for j in range(5)]\n",
+        "\n",
+        "# Generates a random circuit on the provided qubits.\n",
+        "circuit = cirq.experiments.random_rotations_between_grid_interaction_layers_circuit(\n",
+        "    qubits=qubits, depth=16)\n",
+        "\n",
+        "# Simulate the circuit with Cirq and print the runtime.\n",
+        "cirq_simulator = cirq.Simulator()\n",
+        "cirq_start = time.time()\n",
+        "cirq_results = cirq_simulator.simulate(circuit)\n",
+        "cirq_elapsed = time.time() - cirq_start\n",
+        "print(f'Cirq runtime: {cirq_elapsed}')\n",
+        "print()\n",
+        "\n",
+        "# Simulate the circuit with qsim and print the runtime.\n",
+        "qsim_simulator = qsimcirq.QSimSimulator()\n",
+        "qsim_start = time.time()\n",
+        "qsim_results = qsim_simulator.simulate(circuit)\n",
+        "qsim_elapsed = time.time() - qsim_start\n",
+        "print(f'qsim runtime: {qsim_elapsed}')\n"
+      ],
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cirq runtime: 6.540446519851685\n",
+            "\n",
+            "qsim runtime: 0.11791229248046875\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OUjhKoRVZGKZ",
+        "colab_type": "text"
+      },
+      "source": [
+        "qsim performance can be tuned further by passing options to the simulator constructor. These options use the same format as the qsim_base binary - a full description can be found in the qsim [usage doc](https://github.com/quantumlib/qsim/blob/master/docs/usage.md)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "yE6ZXAKzZL5P",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        },
+        "outputId": "1b833005-f23a-42ea-a7d7-b57624bbd3cf"
+      },
+      "source": [
+        "# Use eight threads to parallelize simulation.\n",
+        "options = {'t': 8}\n",
+        "\n",
+        "qsim_simulator = qsimcirq.QSimSimulator(options)\n",
+        "qsim_start = time.time()\n",
+        "qsim_results = qsim_simulator.simulate(circuit)\n",
+        "qsim_elapsed = time.time() - qsim_start\n",
+        "print(f'qsim runtime: {qsim_elapsed}')"
+      ],
+      "execution_count": 8,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "qsim runtime: 0.12204647064208984\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LgWOEMLKutsq",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Distributed execution\n",
```

Comment:
I think you should put this under an "Advanced" section since most people will not be using the "Distributed" portion of qsim, especially if they are running Cirq on their laptop.